backend/otone_client.py

```diff
@@ -252,25 +282,26 @@ def periodically_send_ip_addresses():
 
     subscriber = Subscriber(session_factory, loop)
     publisher = Publisher(session_factory)
-    
+
 
     while (crossbar_status == False):
         try:
-            logging.info('trying to make a connection...')
+            logger.info('trying to make a connection...')
             make_a_connection()
         except KeyboardInterrupt:
             crossbar_status = True
         except:
             #raise
             pass
         finally:
-            logging.info('error while trying to make a connection, sleeping for 5 seconds')
-            time.sleep(5)
+            if not keep_backend_running and disconnect_counter>disconnect_count_threshold:
```

Comment:
this section of code goes above the [PEP8 79 line chars recommendation](https://www.python.org/dev/peps/pep-0008/#maximum-line-length)..

I would prob format it to this,

```
            if (
                    not keep_backend_running and
                    disconnect_counter>disconnect_count_threshold
            ):
                # kill here
                sys.exit()
            else:
                disconnect_counter += 1
                logger.info(
                    'error while trying to make a connection, sleeping for 5 '
                    'seconds (attempt {})'.format(disconnect_counter)
                )
```

Some pythonistas don't agree with the 79 char limit --- but we can talk about that as a team and update our styleguide.. I personally like it; makes it easier to have split sessions on a mac when in iterm...